environment.go

```diff
@@ -31,5 +33,46 @@ func (r *EnvironmentVariableResolver) Resolve(keys []string, _ ConfigMap) (val i
 	if len(r.Prefix) > 0 {
 		keysCopy = append([]string{r.Prefix}, keysCopy...)
 	}
-	return os.LookupEnv(strings.Join(toUpperStringSlice(keysCopy), "_"))
+
+	var translator KeyTranslator
+	if r.Translator == nil {
+		// If we weren't given a specific translator to use, then use the
+		// default translator.
+		translator = DefaultEnvironmentVariableKeyTranslator
+	} else {
+		// Otherwise, use the translator that was provided when this resolver
+		// was created.
+		translator = r.Translator
+	}
+
+	return os.LookupEnv(toEnvironmentVariable(keysCopy, translator))
+}
+
+// The DefaultEnvironmentVariableKeyTranslator is the default KeyTranslator
+// used by the EnvironmentVariableResolver.
+//
+// It is responsible for mapping arbitrary input keys to an environment
+// variable to lookup. This is done by converting all characters to upper case
+// and replacing any hyphens with underscores.
+func DefaultEnvironmentVariableKeyTranslator(b byte) byte {
```

Comment:
If this is the default behavior, could we have a way in which venom could be told to allow the variable key to be set as desired without providing a `KeyTranslator`?

Reply:
Yeah, it probably makes sense to provide some kind of noop translator as well, just for ease of use. I can toss one of those together